recipes/nb_conda_kernels/build.sh

```diff
@@ -0,0 +1 @@
+"${PYTHON}" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I have run into issues with the resolvability of python there, but we'll give it a shot!